api/src/pcapi/scripts/delete_special_event/main.py

```diff
@@ -0,0 +1,24 @@
+from pcapi.app import app
+from pcapi.core.operations import models
+from pcapi.repository.session_management import atomic
+
+
+ID_TO_DELETE = (15, 16)
+
+
+@atomic()
+def delete_events() -> None:
+    response_query = models.SpecialEventResponse.query.filter(models.SpecialEventResponse.eventId.in_(ID_TO_DELETE))
+    models.SpecialEventAnswer.query.filter(
+        models.SpecialEventAnswer.questionId.in_(response_query.with_entities(models.SpecialEventResponse.id))
```

Comment:
Je pense que tu voulais écrire : 
```suggestion
        models.SpecialEventAnswer.responseId.in_(response_query.with_entities(models.SpecialEventResponse.id))
```

Reply:
en effet, je me suis fait avoir par la sandbox (ou les deux ont la même valeur)